
Requirement already satisfied: chardet in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (5.2.0)
[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: python.exe -m pip install --upgrade pip
Failed with encoding: utf-8 - 'utf-8' codec can't decode byte 0xad in position 73: invalid start byte
Successfully read the CSV file with encoding: latin1
                                              domain   ranking mld_res  \
0  nobell.it/70ffb52d079109dca5664cce6f317373782/...  10000000     1.0
1  www.dghjdgf.com/paypal.co.uk/cycgi-bin/webscrc...  10000000     0.0
2  serviciosbys.com/paypal.cgi.bin.get-into.herf....  10000000     0.0
3  mail.printakid.com/www.online.americanexpress....  10000000     0.0
4  thewhiskeydregs.com/wp-content/themes/widescre...  10000000     0.0
  mld.ps_res  card_rem  ratio_Rrem  ratio_Arem  jaccard_RR  jaccard_RA  \
0        0.0      18.0  107.611111  107.277778         0.0         0.0
1        0.0      11.0  150.636364  152.272727         0.0         0.0
2        0.0      14.0   73.500000   72.642857         0.0         0.0
3        0.0       6.0  562.000000  590.666667         0.0         0.0
4        0.0       8.0   29.000000   24.125000         0.0         0.0
   jaccard_AR  jaccard_AA jaccard_ARrd jaccard_ARrem  label
0         0.0         0.0          0.8      0.795729    1.0
1         0.0         0.0            0      0.768577    1.0
2         0.0         0.0            0      0.726582    1.0
3         0.0         0.0            0       0.85964    1.0
4         0.0         0.0            0      0.748971    1.0
C:\Users\PC\AppData\Local\Temp\ipykernel_17048\3038593440.py:6: DtypeWarning: Columns (1,2,3,11,12) have mixed types. Specify dtype option on import or set low_memory=False.
  data = pd.read_csv(r"C:\Users\PC\Documents\MLCodes\urlset.csv", encoding=encoding, on_bad_lines='skip')
                                              domain   ranking mld_res  \
0  nobell.it/70ffb52d079109dca5664cce6f317373782/...  10000000     1.0
1  www.dghjdgf.com/paypal.co.uk/cycgi-bin/webscrc...  10000000     0.0
2  serviciosbys.com/paypal.cgi.bin.get-into.herf....  10000000     0.0
3  mail.printakid.com/www.online.americanexpress....  10000000     0.0
4  thewhiskeydregs.com/wp-content/themes/widescre...  10000000     0.0
  mld.ps_res  card_rem  ratio_Rrem  ratio_Arem  jaccard_RR  jaccard_RA  \
0        0.0      18.0  107.611111  107.277778         0.0         0.0
1        0.0      11.0  150.636364  152.272727         0.0         0.0
2        0.0      14.0   73.500000   72.642857         0.0         0.0
3        0.0       6.0  562.000000  590.666667         0.0         0.0
4        0.0       8.0   29.000000   24.125000         0.0         0.0
   jaccard_AR  jaccard_AA jaccard_ARrd jaccard_ARrem
0         0.0         0.0          0.8      0.795729
1         0.0         0.0            0      0.768577
2         0.0         0.0            0      0.726582
3         0.0         0.0            0       0.85964
4         0.0         0.0            0      0.748971
domain            object
ranking           object
mld_res           object
mld.ps_res        object
card_rem         float64
ratio_Rrem       float64
ratio_Arem       float64
jaccard_RR       float64
jaccard_RA       float64
jaccard_AR       float64
jaccard_AA       float64
jaccard_ARrd      object
jaccard_ARrem     object
dtype: object
[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: python.exe -m pip install --upgrade pip
Requirement already satisfied: wandb in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (0.17.4)
Requirement already satisfied: click!=8.0.0,>=7.1 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (8.1.7)
Requirement already satisfied: docker-pycreds>=0.4.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (0.4.0)
Requirement already satisfied: gitpython!=3.1.29,>=1.0.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (3.1.43)
Requirement already satisfied: platformdirs in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (3.9.1)
Requirement already satisfied: protobuf!=4.21.0,<6,>=3.19.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (5.27.2)
Requirement already satisfied: psutil>=5.0.0 in c:\users\pc\appdata\roaming\python\python311\site-packages (from wandb) (5.9.8)
Requirement already satisfied: pyyaml in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (6.0.1)
Requirement already satisfied: requests<3,>=2.0.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (2.32.3)
Requirement already satisfied: sentry-sdk>=1.0.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (2.10.0)
Requirement already satisfied: setproctitle in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (1.3.3)
Requirement already satisfied: setuptools in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from wandb) (65.5.0)
Requirement already satisfied: colorama in c:\users\pc\appdata\roaming\python\python311\site-packages (from click!=8.0.0,>=7.1->wandb) (0.4.6)
Requirement already satisfied: six>=1.4.0 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from docker-pycreds>=0.4.0->wandb) (1.16.0)
Requirement already satisfied: gitdb<5,>=4.0.1 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from gitpython!=3.1.29,>=1.0.0->wandb) (4.0.11)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from requests<3,>=2.0.0->wandb) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from requests<3,>=2.0.0->wandb) (3.7)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from requests<3,>=2.0.0->wandb) (2.2.2)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from requests<3,>=2.0.0->wandb) (2024.7.4)
Requirement already satisfied: smmap<6,>=3.0.1 in c:\users\pc\appdata\local\programs\python\python311\lib\site-packages (from gitdb<5,>=4.0.1->gitpython!=3.1.29,>=1.0.0->wandb) (5.0.1)
[34m[1mwandb[39m[22m: [33mWARNING[39m Calling wandb.login() after wandb.init() has no effect.